section3/reflection.md

```diff
@@ -1,13 +1,24 @@
 ## Section 3 Reflection
 
 1. In Section 3, we removed even more supports and pushed you to learn a bit more independently, read through technical documentation, and use your resources more. How did that feel?
+   It felt nice to lean on my own knowledge and try out different syntax to see what failed or worked within javascript.
 
 2. Think of a web or mobile application you use regularly. Where is an example of a place that the developers probably use Arrays? Explain.
+   Within my email the list of emails is probably an array.  It is a large list of
+   my emails, and each email can be clicked into and even have different actions done to it.  I bet it is an array of objects that has these functionalities,  and a new index gets added as I get other emails.
 
 3. In your own words, explain what an Array method is.
 
+   An array method is a function that can be used on the Array you're using.  It is similar to when we do addition within a log function, but these methods are already defined and can be called on the Array we created.
+
 4. What is the purpose of a loop? When we would use a loop in conjunction with array vs. without?
+   A loop is used when you want to perform a certain function or change multiple times, or you want to iterate and test variables in an array.  You might use a for loop with an array when you want to use all the variables within that array, you might be making several messages and concatenating the variables within those messages.  You wouldn't necessarily use an array to output the result of one integer having another integer added to it _n_ times.
 
 5. If you had to teach a 5 year old about loops, how would you explain it?
 
+   A _for_ loop in coding is when you have a carton of eggs, and you need 6 for your recipe, so you need to crack an egg into the bowl 6 times. You have to tell your robot helper to do things one step at a time. We tell him to track how many eggs we've cracked starting at 0 and it must be less than 6.  Then he must crack the egg into the bowl, and add to the times he added to the bowl by one. He checks to make sure he hasn't gone to 6 then cracks an egg again. Once he gets to 6 he stops.
+
 6. What questions do you still have about Arrays and loops?
+
+   When you define how many indicies a constructor Array starts with, does javascript
```

Comment:
I'm not 100% clear what you're asking?? But I'll take a guess - if I declare 

```
names = ["Mo", "Blue", "Amy"]
```

"Mo" is set to 0 being the first element, "Blue" is 1, etc.